tests/test_sessions_api.py

```diff
@@ -283,6 +283,136 @@ def test_minute_granularity_range(self, get_project_id: Callable[[], int]) -> No
         }
 
 
+class TestSessionsApi(BaseSessionsMockTest, BaseApiTest):
```

Comment:
Are these tests related to the column change?

Reply:
how would it fail if the column change wasn't made

Reply:
It won't on it's own. This is a small part of the larger initiative to start validating columns. I'm breaking that work down into more reviewable chunks. Ultimately I will turn on column validation for this entity, and at that point this test would start failing without the check.

Reply:
So when validation is turned on this test is expected to fail? On what line? 

Also, is the generating of data necessary for the validation test?

Reply:
> So when validation is turned on this test is expected to fail? On what line?

This PR both introduces an error and fixes it. Turning the column validation to `WARN` would cause warning messages to get sent to Sentry, except that I added `bucketed_started` to the Entity data model, which should fix the warnings. So no, the test should pass and continue to pass. I added it because we weren't testing that code path at all.

> Also, is the generating of data necessary for the validation test?

I want to make sure that `bucketed_started` was returning bucketed data correctly.